tailcall-tracker/src/tracker.rs

```diff
@@ -4,8 +4,16 @@ use super::Result;
 use crate::check_tracking::check_tracking;
 use crate::event::Event;
 
-const API_SECRET: &str = "GVaEzXFeRkCI9YBIylbEjQ";
-const MEASUREMENT_ID: &str = "G-JEP3QDWT0G";
+const API_SECRET: &str = match option_env!("GA_API_SECRET") {
+    Some(val) => val,
+    None => "dev",
+};
+
+const MEASUREMENT_ID: &str = match option_env!("GA_MEASUREMENT_ID") {
+    Some(val) => val,
+    None => "dev",
+};
```

Comment:
CI also needs to be updated with the env variables being set.

Reply:
@ridoyshak35